README.adoc

```diff
@@ -36,30 +38,238 @@ include::https://raw.githubusercontent.com/spring-guides/getting-started-macros/
 
 
 [[initial]]
-== Create a resource controller
+== Add required dependencies
+
+To use the Spring Integration GCP Pub/Sub channel adapters, you'll need to pull in the
+`spring-integration-gcp` dependency. We also strongly recommend the Spring Cloud GCP Pub/Sub Boot
+starter, `spring-cloud-gcp-starters-pubsub`, to deal with all the object instantiation and auto
+configuration.
+
+At the time of writing, the Spring Cloud GCP libraries are still in Alpha stage. As such, they are
```

Comment:
Suggestion:
Currently, the Spring Cloud GCP libraries are still in Alpha, so they are only published to the Spring Snapshots Maven repository.